packages/core/strapi/lib/core/loaders/plugins/get-enabled-plugins.js

```diff
@@ -26,12 +26,17 @@ const toDetailedDeclaration = declaration => {
   }
   if (has('resolve', declaration)) {
     let pathToPlugin = '';
-    try {
-      pathToPlugin = dirname(require.resolve(declaration.resolve));
-    } catch (e) {
-      if (existsSync(declaration.resolve) && statSync(declaration.resolve).isDirectory()) {
-        pathToPlugin = declaration.resolve;
-      } else {
+
+    if (isPath(declaration.resolve)) {
```

Comment:
How do we handle path like `C:\some-dir\file` ? 

Would it work if we did: try a simple require.resolve and then a require.resolve with the path.resolve(strapi.dir, value) ?

Reply:
Good point, I'll think of that

Reply:
Figuring out if the string is more likely a path or a npm module seems too tricky.
For the moment I went back to the system of before : trying to resolve as module first and, if it doesn't find it, try to resolve as a path (append with strapi.dir if necessary).
Conflict should be rare.
Do you have a better idea?